diff -up awstats-7.9/wwwroot/cgi-bin/awstats.model.conf.defaultconf awstats-7.9/wwwroot/cgi-bin/awstats.model.conf
--- awstats-7.9/wwwroot/cgi-bin/awstats.model.conf.defaultconf	2021-08-19 04:00:58.000000000 +0200
+++ awstats-7.9/wwwroot/cgi-bin/awstats.model.conf	2023-09-15 14:37:38.128564435 +0200
@@ -47,7 +47,7 @@
 # If there are several log files from load balancing servers :
 # Example: "/pathtotools/logresolvemerge.pl *.log |"
 #
-LogFile="/var/log/httpd/mylog.log"
+LogFile="/var/log/httpd/access_log"
 
 
 # Enter the log file type you want to analyze.
@@ -219,7 +219,7 @@ DynamicDNSLookup=0
 # Example: "C:/awstats_data_dir"
 # Default: "."          (means same directory as awstats.pl)
 #
-DirData="."
+DirData="/var/lib/awstats"
 
 
 # Relative or absolute web URL of your awstats cgi-bin directory.
@@ -228,7 +228,7 @@ DirData="."
 # Example: "/awstats"
 # Default: "/cgi-bin"   (means awstats.pl is in "/yourwwwroot/cgi-bin")
 #
-DirCgi="/cgi-bin"
+DirCgi="/awstats"
 
 
 # Relative or absolute web URL of your awstats icon directory.
@@ -238,7 +238,7 @@ DirCgi="/cgi-bin"
 # Example: "../icon"
 # Default: "/icon" (means you must copy icon directories in "/mywwwroot/icon")
 #
-DirIcons="/icon"
+DirIcons="/awstats/icon"
 
 
 # When this parameter is set to 1, AWStats adds a button on the report page to
@@ -836,7 +836,7 @@ DecodeUA=0
 # Possible value: URL of javascript tracker file added in your HTML code.
 # Default: "/js/awstats_misc_tracker.js"
 #
-MiscTrackerUrl="/js/awstats_misc_tracker.js"
+MiscTrackerUrl="/awstats/js/awstats_misc_tracker.js"
 
 
 # AddLinkToExternalCGIWrapper can be used to add a link to a wrapper script
